onnx/defs/experiments/experiments_functions.cc

```diff
@@ -9,161 +9,155 @@ namespace ONNX_NAMESPACE {
 using SupportType = OpSchema::SupportType;
 using SupportType = ONNX_NAMESPACE::OpSchema::SupportType;
 
-static FunctionProto BuildMVN() {
-  FunctionProto func;
-  func.set_name("MeanVarianceNormalization");
-  func.set_doc_string(
-      "A MeanVarianceNormalization Function: Perform mean variance normalization "
-      "on the input tensor X using formula: <br/> ``` (X-EX)/sqrt(E(X-EX)^2) ``` <br/><br/>"
-      "<b>INPUT: </b>X(float/float16/double) with shape [N,C,W,H] or N-D shape <br/><br/>"
-      "<b>ATTRIBUTE: </b><br/>&nbsp;&nbsp;&nbsp;&nbsp;<tt>axes: </tt>will be passed to ReducedMean "
-      "Ops. Use [0,2,3] (without C axis for N-D cases) for calculating means and variances "
-      "along channels. Two variables with the same C-coordinate are associated "
-      "with the same mean and variance. Use [0,1,2,3] (with C axis) to calculate "
-      "global mean and global variance with all variables sharing the same mean/variance.<br/>"
-      "&nbsp;&nbsp;&nbsp;&nbsp;(The KeepDims attribute in ReducedMean is set to true for calculation)<br/>"
-      "<br/><b>OUTPUT: </b>X_MVN(float/float16/double) with the same shape as input X<br/>");
-  func.set_since_version(9);
-  func.add_input("X");
-  func.add_output("X_MVN");
-  func.add_attribute("axes");
-  func.set_status(OperatorStatus::STABLE);
-
-  NodeProto* initial_node0 = func.add_node();
+static std::vector<NodeProto> BuildMVN() {
```

Comment:
I am slightly concerned that there may be some extra copying going on here: but may be the vectors will be "moved" cheaply by the compiler (avoiding redundant copies)?